.circleci/config.yml

```diff
@@ -11,6 +11,7 @@ jobs:
       docker_layer_caching: true
     steps:
       - checkout
+      - run: make lint
```

Comment:
this should be a different job ideally, can you instead add a new `lint` job with the `make lint` step and add to the `workflow.flow.jobs` lower in the file

Reply:
Ok, I fix it !